src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java

```diff
@@ -68,6 +68,18 @@
  *   &lt;property name=&quot;eachLine&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>violation</p>
+ * <pre>
+ * for(int i=0; i&lt;10; i++){
+ *     //violation(extra spaces after open bracket)
```

Comment:
this comment seems to refer to a different check (spaces after open bracket) and not tab characters

Reply:
it is very hard to show something that is not visible so example of code should be in form like:
```
$ cat Test1.java 
public class Test1 {
	int field; // tab character at the beginning of line
}

$ cat -A Test1.java 
public class Test1 {$
^Iint field; // tab character at the beginning of line$
}$
```